CMakeLists.txt

```diff
@@ -27,6 +27,15 @@ if (NOT DEFINED CYCFI_JSON_ROOT)
   set(CYCFI_JSON_ROOT ../json)
 endif()
 
+set(WINDOWS_GRAPHICS_LIBRARY "native" CACHE STRING "build with windows native gui api or gtk")
+if(WINDOWS_GRAPHICS_LIBRARY STREQUAL "gtk")
+    add_definitions(-D__WINDOWS_WITH_GTK__)
```

Comment:
Avoid double underscores, they are reserved for implementations.

Reply:
ok.